spec/controllers/lambda_callback/document_proof_result_controller_spec.rb

```diff
@@ -20,36 +19,25 @@
     context 'with valid API token' do
       before do
         request.headers['X-API-AUTH-TOKEN'] = AppConfig.env.document_proof_result_lambda_token
-
-        document_capture_session.store_proofing_pii_from_doc(pii)
-
-        proofer_result = ProofingDocumentCaptureSessionResult.new(id: id, pii: pii,
-                                                                  result: idv_result)
-        document_capture_session.store_proofing_result(proofer_result)
+        document_capture_session.store_proofing_pii_from_doc({}) # generates a result_id
```

Comment:
this is the kind of side effect shenanigans I dislike, but all our result controllers seem to use the DocumentCaptureSession in this goofy way